salesforce/tests/rest_api_test.bal

```diff
@@ -115,45 +115,54 @@ function testDeleteRecord() {
 }
 function testGetQueryResult() returns error? {
     log:printInfo("baseClient -> getQueryResult()");
-    string sampleQuery = "SELECT name FROM Account";
-    SoqlResult res = check baseClient->getQueryResult(sampleQuery);
-    assertSoqlResult(res);
-    string nextRecordsUrl = res["nextRecordsUrl"].toString();
-
-    while (nextRecordsUrl.trim() != EMPTY_STRING) {
-        log:printInfo("Found new query result set! nextRecordsUrl:" + nextRecordsUrl);
-        SoqlResult resp = check baseClient->getNextQueryResult(nextRecordsUrl);
-        assertSoqlResult(resp);
-        res = resp;
-    }
+    string sampleQuery = "SELECT Name,Industry FROM Account";
+    stream<record {}, error?> resultStream = check baseClient->getQueryResult(sampleQuery);
+    int count = check countStream(resultStream);
+    test:assertTrue(count > 0, msg = "Found 0 search records!");
+}
+
+@test:Config {
+    enable: true
+}
+function testGetQueryResultWithLimit() returns error? {
+    log:printInfo("baseClient -> getQueryResult()");
+    string sampleQuery = "SELECT Name,Industry FROM Account LIMIT 3";
+    stream<record {}, error?> resultStream = check baseClient->getQueryResult(sampleQuery);
+    int count = check countStream(resultStream);
+    test:assertTrue(count > 0, msg = "Found 0 search records!");
+}
+
+@test:Config {
+    enable: false
+}
+function testGetQueryResultWithPagination() returns error? {
+    log:printInfo("baseClient -> getQueryResult()");
+    string sampleQuery = "SELECT Name FROM Contact";
+    stream<record {}, error?> resultStream = check baseClient->getQueryResult(sampleQuery);
+    int count = check countStream(resultStream);
+    log:printInfo("Number of records", count = count);
+    test:assertTrue(count > 2000, msg = "Found less than 2000 search records!");
```

Comment:
Suggestion
```
test:assertTrue(count >= 2000, msg = "Found less than 2000 search records!");
```

Reply:
Fixed in 8c88af2857fa79a4d714f0fef58d93bab32cb3b9